docs/docs/configuration.md

```diff
@@ -115,7 +115,7 @@ The value of these properties are not persisted as a part of the table metadata.
 
 | Property       | Default  | Description                                                                                                                          |
 | -------------- | -------- |--------------------------------------------------------------------------------------------------------------------------------------|
-| format-version | 2        | Table's format version (can be 1 or 2) as defined in the [Spec](../../spec.md#format-versioning). Defaults to 2 since version 1.4.0. |
+| format-version | 2        | Table's format version (can be 1, 2 or 3) as defined in the [Spec](../../spec.md#format-versioning). Defaults to 2 since version 1.4.0. |
```

Comment:
Maybe we'd better not mention possible values of `format-version` here, since we already link to the source. Otherwise, the list can be too long in the future.